emeis/core/views.py

```diff
@@ -1,8 +1,21 @@
+from django.contrib.auth import get_user_model
+from rest_framework.mixins import RetrieveModelMixin
+from rest_framework.viewsets import GenericViewSet
 from rest_framework_json_api import views
 
 from . import models, serializers
 
 
+class MeViewSet(RetrieveModelMixin, GenericViewSet):
```

Comment:
Couldn't this just be an `@action` endpoint, since it supports only one action anyways? Would make the router a little simpler too

Reply:
I prefer always using viewsets for consistency. But that's my personal preference. Additionally it's unclear if we will allow a user to update it's own record at a later point. It's in a grey area of pre-mature optimization, I know :)

Reply:
I can live with that :)